components/ApplicationDetails/ApplicationDetails.js

```diff
@@ -106,7 +106,8 @@ const ApplicationDetails = ({ data }) => {
               </div>
             </div>
             {/* this is just an if statement that checks if a user session exists */}
-            {session && <Vote voteCount={voteCount} onVote={onVote} applicationId={data._id} />}
+            {session && <Vote voteCount={voteCount} onVote={onVote} applicationId={data._id} isUserlogged={true} />}
+            {!session && <Vote voteCount={voteCount} onVote={onVote} applicationId={data._id} isUserlogged={false} />}
```

Comment:
```suggestion
            <Vote voteCount={voteCount} onVote={onVote} applicationId={data._id} isUserAuthenticated={session?.authenticated} />
```

I'd like to suggest slightly rephrasing this prop to something like `isUserAuthenticated` and use a boolean value from `session` instead of having the component twice.